download/releaseNotes/releaseNotes8.adoc

```diff
@@ -1,17 +1,95 @@
 = Release Notes 8
-:awestruct-description: New and noteworthy, demos and status for OptaPlanner 7.
+:awestruct-description: New and noteworthy, demos and status for OptaPlanner 8.
 :awestruct-layout: releaseNotesBase
 :awestruct-priority: 1.0
 :awestruct-release_notes_version: 8
-:awestruct-release_notes_version_qualifier: Final
+:awestruct-release_notes_version_qualifier: Beta
 
-== New and noteworthy: Engine 8.0.0.Final
+== What is OptaPlanner 8?
 
-=== XML Schema Definition for both solverConfig.xml and benchmarkConfig.xml
+*OptaPlanner 8 is basically OptaPlanner 7 without the deprecated methods*
+and a few minor backwards incompatible changes.
+**Migrating your code from 7 to 8 is easy and quick.**
+Just follow the instructions in link:../upgradeRecipe/upgradeRecipe8.html[the upgrade recipe].
 
-In this release, we are introducing XML Schema Definition for an XML configuration of both the solver and the benchmark.
-IDEs can now offer code completion of the solver and the benchmark configuration.
+As usual, Red Hat offers link:../../product/services.html[enterprise support and consulting] on OptaPlanner 8,
+with the _Red Hat build of OptaPlanner_ in an upcoming release of Red Hat Decision Manager 7
+in the first half of 2021.
+
+== New and noteworthy: Engine 8.0.0.Beta1
+
+=== OptaPlanner quickstarts repository
+
+The new https://github.com/kiegroup/optaplanner-quickstarts[OptaPlanner Quickstarts repository]
+contains beautiful web demo's for multiple use cases.
```

Comment:
"Beautiful"? :-) It may be the first time I'm seeing such a word anywhere around OptaPlanner.

Reply:
Also, "demos".

Reply:
And given everybody has a different opinion on what beautiful is, I would rather avoid the word.

Reply:
Yea, it's marketing talk, not engineering talk. I had originally written "pretty".
The idea was to clearly stand out that we're listening to all the people (20 or so) that gave the feedback "why do your demos use ugly swing? People want to see pretty webapps."

Reply:
> Also, "demos".

I am sure you haven't seen such mistakes before :smile:

Reply:
I made it "pretty" :)
Motivation: the "new and noteworthy" section is to get people excited, to convey the goodness and positive vibes (that this release is definitely entitled to). For those asking about prettier demo's, that will make them recognize we are (finally) listening.
Docs must avoid such frivolous wordings indeed. New and noteworhy aren't docs.